website/docs/components/form/masked-input/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,79 @@
+## Usage
+
+### When to use
+
+- To give users a way to show and hide sensitive information.
+
+- When a form input with the ability to mask its value temporarily is needed.
+
+### When not to use
+
+- To define a password field, use the [Password](/components/form/text-input#password) field instead.
+
+- For general textual input, use the [TextInput](/components/form/text-input) or [Textarea](/components/form/textarea).
+
+
+## Masked input vs Password input
+
+The MaskedInput is a text field designed to protect sensitive or confidential information while still giving users the ability to show or hide the input value as needed.
+
+A password field is a specific type of form control designed for secure entry of passwords. The value is always masked or hidden to prevent others from easily reading the entered characters. Unlike the masked input, the password field has specific attributes to enhance the security and confidentiality of passwords.
```

Comment:
```suggestion
The password input is a specific type of form control designed for the secure entry of passwords. The value is always masked or hidden to prevent others from reading the entered characters easily. Unlike the Masked Input, the password input has specific attributes to enhance the security and confidentiality of passwords.
```